Sources/HTMLString/HTMLString.swift

```diff
@@ -94,31 +94,35 @@ extension String {
     /// | `a` | `a` | Not escaped (alphanumerical) |
     ///
 
-    public mutating func addASCIIEntities() {
+    public func addASCIIEntities() -> String {
         var position: String.Index? = startIndex
         let requiredEscapes: Set<Character> = ["!", "\"", "$", "%", "&", "'", "+", ",", "<", "=", ">", "@", "[", "]", "`", "{", "}"]
 
+        var result = ""
```

Comment:
Same here with the capacity

```suggestion
        var result = String()
        result.reserveCapacity(count)
```

Reply:
I replied above. My opinion is that reserving capacity is not much good as expected. It's slower than no reservation. The test tells us truth.